peer.go

```diff
@@ -1142,30 +1174,27 @@ func (p *peer) handleUpstreamMsg(state *commitmentState, msg lnwire.Message) {
 				p.err <- nil
 				delete(state.clearedHTCLs, htlc.ParentIndex)
 			}
-
 			// TODO(roasbeef): rework log entries to a shared
 			// interface.
 			if htlc.EntryType != lnwallet.Add {
 				continue
 			}
-
 			// If we can't immediately settle this HTLC, then we
 			// can halt processing here.
 			invoice, ok := state.htlcsToSettle[htlc.Index]
 			if !ok {
+				peerLog.Infof("We cannot handle invoice for htlc.Index=%v", htlc.Index)
```

Comment:
I'm not sure if this is worth logging. In any case, it should be a `Debugf` rather than to the info level.